docs/spec.md

```diff
@@ -1706,9 +1706,8 @@ imaginary values, `lhs` and `rhs`, and produces a `result` tensor.
 
 #### Semantics
 
-Concatenates a variadic number of tensors in `inputs` along `dimension`
-dimension in the same order as the given arguments and produces a `result`
-tensor. More formally,
+Concatenates `inputs` along `dimension` dimension in the same order as the given
+arguments and produces a `result` tensor. More formally,
```

Comment:
Rewords "a variadict number of tensors in `foo`" to just "`foo`" to compress wording without losing intent.